game/WiiU3DSMiitomo_miidatafile.ksy

```diff
@@ -0,0 +1,178 @@
+meta:
+  id: WiiU3DSMiitomo_miidatafile

```

Comment:
This isn't going to work:

```
io.kaitai.struct.format.YAMLParseException: /meta/id: invalid meta ID: 'WiiU3DSMiitomo_miidatafile', expected /^[a-z][a-z0-9_]*$/
```

Kaitai Struct doesn't accept uppercase letters in `id`s, because it might need to translate the identifier depending on the target language. For example, class names are by convention written in `CamelCase` in many languages, so a KSY named `some_long_name` will be translated to a class named `SomeLongName`. This kind of translation becomes more complicated and unclear if the original name already has uppercase letters in it, so Kaitai Struct just forbids uppercase letters in identifiers.

Reply:
> Kaitai Struct doesn't accept uppercase letters in `id`s, because it might need to translate the identifier depending on the target language.

Yes, the reason is described in https://github.com/kaitai-io/kaitai_struct/issues/148.